images/scripts/rhel.setup

```diff
@@ -284,6 +284,12 @@ if [ "${IMAGE#rhel-[89]}" != "$IMAGE" ] || [ "${IMAGE#centos-[89]}" != "$IMAGE"
     TEST_PACKAGES="$TEST_PACKAGES python3-tracer"
 fi
 
+# Centos 10 has audit-rules but doesn't install them by default. Let's
+# test with them explicitly.
+if [ "${IMAGE#centos-10*}" != "$IMAGE" ]; then
+    TEST_PACKAGES="$TEST_PACKAGES audit-rules"
```

Comment:
I'm not a fan of this, it changes the default install behaviour quite considerably. If it's not installed by default, and we want it, then this should become a dependency of `cockpit-selinux` (or Recommends: of `cockpit-system` on RHEL). I actually prefer the test skip over this.

Reply:
Hmm, I can't say I have an opinion. Skipping is fine, but changing our package dependencies needs a reason, and I can't provide any right now.